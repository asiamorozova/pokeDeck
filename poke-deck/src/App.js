import React, { Component } from 'react';
import logo from './logo.svg';
import './App.css';
//import pokemon from './data.js';
//import Container from './pokelist.js';
import Map from './map.js';
import request from 'superagent';
import Header from './Header.js';
import ExploreApp from './exploreDeck.js';


class App extends Component {




  render() {
    return (
      <main>
        <ExploreApp />

      </main>
    )
  }
}
export default App;