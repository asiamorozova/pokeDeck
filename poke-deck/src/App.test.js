import React from 'react';
import renderer from 'react-test-renderer';
import App from './App.js';

test('This render App', () => {
  const app = renderer
    .create(<App />)
    .toJSON();
  expect(app).toMatchSnapshot();
});
