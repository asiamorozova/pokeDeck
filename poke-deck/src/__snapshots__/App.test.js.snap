// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`This render App 1`] = `
<main>
  <img
    src="https://www.pngkey.com/png/detail/32-329404_frames-illustrations-hd-download-the-vector-logo-pokemon.png"
  />
  <ul />
</main>
`;

exports[`This render Header 1`] = `
<img
  src="https://www.pngkey.com/png/detail/32-329404_frames-illustrations-hd-download-the-vector-logo-pokemon.png"
/>
`;
